### DISCUSSION TITLE
Custom Sprites

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#127] [Q&A]
Can I use (custom) sprites from sources like the sprinters resource and deviantart in Sonic Worlds Next?

**Comment by Techokami** [comment]
Why do you keep making new threads to ask simple questions? Sonic Worlds Next is a *framework* that you use to create the game you want to make, with whatever assets you want.

**Reply by WimpyKidFan2007** [reply]
Because I'm very curious.

**Reply by Techokami** [reply]
But there's no need to make *multiple threads* to ask your questions! Just... keep all your questions to one thread.

**Comment by DimensionWarped** [comment]
Yeah, you should stop thinking of it like Mario Maker. Anything you can make with Godot, you can make *starting* from this engine. With enough effort you can replace every single element of what is included with something else. The only question is how much of what is included you leave in.
